README.md

```diff
@@ -1 +1,140 @@
+[![Build Status](https://travis-ci.com/oasis-roles/vmware_provision.svg?branch=master)](https://travis-ci.com/oasis-roles/vmware_provision)
 
+vmware_provision
+===========
+
+Provisions VMWare hosts and adds them to inventory groups. This role also
+provides the information for newly provisioned instances through the variable
+`provisioned_vms` for use elsewhere in the playbook
```

Comment:
Should this var be namedspaced?  I'm curious if it would ever be used outside of this role.

Reply:
:+1: should be `vmware_provisioned_vms`

Reply:
I added a PR here:

https://github.com/andrew103/vmware_provision/pull/1